src/status_im/contexts/wallet/create_account/view.cljs

```diff
@@ -124,7 +143,17 @@
            :customization-color @account-color
            :on-auth-success     (fn [entered-password]
                                   (if new-keypair
-                                    (js/alert "Feature under development")
+                                    (rf/dispatch
```

Comment:
hmm, looking at this event handler I think we need to revisit how we are working with the create-account page. I was discussing this with @briansztamfater today as the send flows also have the same issue and we need to ensure the code is easier to understand and easy to update/ extend.

In the case of create account there are some other alternative flows that we also need to consider (private key backup etc)

At the moment we seem to be internally handling the checks for what flow this is but probably we should create wrapper pages. Let's have a call tomorrow or Wednesday to discuss as probably we should refactor this page first before adding further code to it and properly plan out how we handle these features.

Reply:
@J-Son89 I agree, we can simplify. Maybe can have a single event to dispatch from the view, and then that event dispatches the right flow of events from inside the events file